### PR TITLE
Fixes makeshift grinders

### PR DESCRIPTION
**Pull request by Lyroy**
## About The Pull Request
<details>
<summary>
	Makeshift grinder insertion code was using a lazylen on something that wasn't a list and so everything that wasn't a reagent_container wasn't grindable. I just took the check from the regular grinder instead.
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:
fix: makeshift grinders can properly grind non-food items now
/:cl:

### CODE REVIEW

**Comment by ThePainkiller** [comment]
Does this also make it so that they can grind material sheets into liquids?

File: code/modules/reagents/machinery/grinder.dm [hunk 1]
```diff
@@ -436,7 +436,7 @@
 	if(istype(I, /obj/item/reagent_containers))
 		var/obj/item/reagent_containers/container = I
 		if(!container.standard_pour_into(user, src)) . = ..()
-	else if (LAZYLEN(I.reagents)) . = ..()
+	else if (I.reagents && I.reagents.total_volume) . = ..()
```
Comment (r4d6) [inline code/modules/reagents/machinery/grinder.dm:439]:
```suggestion
	else if (I.reagents?.total_volume) . = ..()
```
This way if reagents is null then it won't cause a runtime and just return null, which is the same as the false & the previous I.reagents check.

**Comment by Lyroy** [comment]
> Does this also make it so that they can grind material sheets into liquids?

Nope, only the grinder machines can do that still.

**Comment by ThePainkiller** [comment]
> Nope, only the grinder machines can do that still.

Disappointing. We need a ghetto way to get liquid iron tbh

**Comment by Trilbyspaceclone** [comment]
Do to it being a bug fix no need to wait on it

**Pull request merged**